scipy/stats/tests/test_stats.py

```diff
@@ -1179,6 +1179,12 @@ def test_weightedtau():
     tau, p_value = stats.weightedtau(x, y)
     assert_approx_equal(tau, -0.56694968153682723)
 
+def test_segfault_issue_9710():
+    # https://github.com/scipy/scipy/issues/9710
+    # This test was created to check  segfault
+    stats.weightedtau([1], [1.0])
+    stats.weightedtau([1], [1.0])
```

Comment:
Suggest adding a comment that the 2nd call triggers the seg fault, to make it easier for future readers of the test.

Reply:
Ok
Added comments to clarify second call and test with np.nan